CHANGELOG.md

```diff
@@ -1,3 +1,83 @@
+# Pelion Edge 2.4.0 - 24th June 2021
+
+### New features
+
+- Updated [LmP version to 81](https://foundries.io/products/releases/81/).
+- Secure Edge container application with Parsec and the Trust Platform Module (TPM) v2.0.
+   - Access TPM resource through Parsec APIs from within the container application.
+   - The storage of secure assets, such as keys, is separated on a per-client basis: Assets created by one client can't be accessed by another.
+   - Use `securityContext` in the Pod specifications to restrict the privileges and access control of an application to system resources.
+- [edge-core] Updated Edge Core to [0.18.0](https://github.com/PelionIoT/mbed-edge/releases/tag/0.18.0).
+   - Start edge-core service `After=network-online.target` which is after gateway acquires routeable IP address.
+   - Restrict access to mbed configuration `/userdata/mbed`, setting permissions to 700 with owner=root.
+   - Added BYOC_MODE build flow. This enables you to inject the certificates into the image at runtime rather than compile time hence enabling you to create generic developer builds.
+- [edge-tool] Added [Edge tool v0.2.0](https://github.com/PelionIoT/mbed-edge/tree/master/edge-tool).
+   - Installed only when build with `BYOC_MODE=ON`.
+   - Converts the development certificate to CBOR configuration object which is then provided to Edge Core as command line argument.
+- [TBC] LmP RPi3/4 support.
+- [TBC] Add support for Hummingboard Solid with booting from SD card.
+- [TBC] Network policy
+   - allows kubelet and kube-router to function without external network. Supports offline cache mode
+- [TBC] UZ - align wks.in file usage with i.mx8
+- [TBC] Change default image file name for MFG tool - `MFGTOOL_FLASH_IMAGE = "console-image-lmp"`.
+- [TBC] Removed meta-arm-autonomy layer.
+- [TBC] add dependency to pc-ble-driver into config, default off
+- Enable `wifi` by default for all targets.
+- Added imx8mmsolidrun meta layer.
+- [TBC] Added support for full image update.
+- [TBC] Prevent duplicate deployment. 
+   - Modifed to check if the commit has been deployed before making the deployment. This will prevent an issue where a previous deploy can get over-written which in turn would break the rollback functionality.
+- [parsec] Upgraded parsec-se-driver to 0.5.0, parsec-service to 0.7.2 and parsec-tool to 0.3.0.
+   - Set the parsec sock directory permission to 0750
+- [TBC] Simplified the partition layout - https://github.com/PelionIoT/meta-mbed-edge/pull/51 
+- [TBC] Generic `mx8mm` support - Instead of using the imx8mmevk target, let's use mx8mm to generalize the support as the current changed should run on all targets. Generalizing thus to the SoC level target `mx8mm`.
+- [TBC] Removed `networkmanager-nmtui` and instead installed `networkmanager-nmcli`.
+- [TBC] use miniuart bt dtoverlay to enable bluetooth
+   - just use standard console and ble settings
+- [TBC] Use standard tty config, fixes ble
```

Comment:
```suggestion
- [Other] Change default image file name for MFG tool - `MFGTOOL_FLASH_IMAGE = "console-image-lmp"`.
- [Space conservation] Removed meta-arm-autonomy layer.
- [Upgrades]
    - Added support for full image update.
    - Prevent duplicate deployment. Modifed to check if the commit has been deployed before making the deployment. This will prevent an issue where a previous deploy can get over-written which in turn would break the rollback functionality.
- [parsec] Upgraded parsec-se-driver to 0.5.0, parsec-service to 0.7.2 and parsec-tool to 0.3.0.
   - Set the parsec socket directory permission to 0750.
- [Image] Simplified the partition layout - https://github.com/PelionIoT/meta-mbed-edge/pull/51 .
- [build process] Generic `mx8mm` support - Instead of using the imx8mmevk target, let's use mx8mm to generalize the support as the current changed should run on all targets. Generalizing thus to the SoC level target `mx8mm`.
- [OS general] 
    - Removed `networkmanager-nmtui` and instead installed `networkmanager-nmcli`.
    - Enable `wifi` by default for all targets.
```